.github/workflows/build.yaml

```diff
@@ -630,3 +630,15 @@ jobs:
       - name: Output
         if: ${{ ! cancelled() }}
         run: cat target/rat.txt || echo "Unable to print output"
+
+  container-images:
+    name: Container Images
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build Debian image
+        uses: docker/build-push-action@v2
+        with:
+          file: dockerfiles/Dockerfile-debian
+          context: .
```

Comment:
I don't insist on you adding it in this PR, but the dispatch project is missing a `.dockerignore` file. This means that people who have a `build` directory in their checkout and want to build the image will suffer the time penalty while docker copies the entire directory (with all logs from the test runs, ...) into the docker build context.

Good .dockerignore for dispatch can look something like this, https://github.com/skupperproject/skupper-router/blob/9017643e419af523236f643d54b0aca42f109e57/.dockerignore

Reply:
Added, hope it's fine.